BICCN-policies/DRAFT Citation and identifier policy.md

```diff
@@ -0,0 +1,49 @@
+# Policy regarding support of data and resource citation and use of persistent identifiers
+
+## For data submitters/authors
+
+### Persistent identifiers + metadata for BICCN datasets
+- BICCN consortium members will submit their data to an approved BRAIN repository for publication.  
+- Datasets must include a standard set of metadata specified by the BICCN (TODO:  Get list from Carol)
+- Repositories will issue persistent identifiers (PIDs) required by many journals for referencing a dataset in a publication. 
+- If submitted data are to be included in a manuscript, make sure that you consult with the repository to understand how long it takes for data ingest, validation and PID assignment. (TODO:  Reference charter location)
+
+### Data and resource citation policy for BICCN
+
```

Comment:
Should we add a step in here to request that creators of new resources register their tool or resource with RRID? I think the tools will be better catalogued if the entries are drafted by the creators themselves.